src/palindrom-dom.js

```diff
@@ -10,29 +10,43 @@
    * @param {Object} [options] map of arguments. See README.md for description
    */
   var PalindromDOM = function (options){
-    options || (options={});
+    if (typeof options !== "object") {
+            throw new Error("'options' is not an object");
+      }
+    if (!options.remoteUrl) {
+          throw new Error('remoteUrl is required');
+    }
     var onDataReady = options.callback;
     this.element = options.listenTo || document.body;
     var clickHandler = this.clickHandler.bind(this);
     this.historyHandler = this.historyHandler.bind(this);
+    this.historyHandlerDeprecatedMessage = function () {
+      console.warn("`puppet-redirect-pushstate` event is deprecated, please use `palindrom-redirect-pushstate`, if you're using `puppet-redirect`, please upgrade to `palindrom-redirect`");
+      this.historyHandler.bind(this)();
```

Comment:
I'd rather move this `.bind(this)` below, not to create new function every time it's called, or just use closure.